metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/KafkaComponent.java

```diff
@@ -137,13 +141,20 @@ public String getBrokerList()  {
     return createProducer(String.class, byte[].class);
   }
 
+  public AdminClient createAdminClient() {
+    Map<String, Object> adminConfig = new HashMap<>();
+    adminConfig.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, getBrokerList());
+    AdminClient adminClient = KafkaAdminClient.create(adminConfig);
```

Comment:
Based on my reading of the [Javadocs](https://kafka.apache.org/20/javadoc/org/apache/kafka/clients/admin/KafkaAdminClient.html), we should not refer directly to `KafkaAdminClient`.  Just refer to `AdminClient` instead.
```
AdminClient adminClient = AdminClient.create(adminConfig);
```

> KafkaAdminClient
> The default implementation of AdminClient. An instance of this class is created by invoking one of the create() methods in AdminClient. Users should not refer to this class directly. The API of this class is evolving, see AdminClient for details.

Reply:
Great catch.  I will change this.

Reply:
Done